Sources/Extensions/String+OneLogin.swift

```diff
@@ -0,0 +1,9 @@
+extension String {
```

Comment:
I'm not sure we should be extending `String` in this way, it's a bit of an anti-pattern / violation of single-responsibility principle.

If you want these static vars, I'd add them to the Environment directly:
`AppEnvironment.oneLoginClientID` and then make the `string(for...)` method private.

Reply:
Gotcha 🙂, making the `string(for:...)` method private would render it unusable for [URL+Extensions](https://github.com/alphagov/di-mobile-ios-onelogin-app/blob/37465347e1c635c452594a66fe751146e5b479f7/Sources/Extensions/URL%2BOneLogin.swift)
Should I move these into `AppEnvironment` as well?